jujugui/static/gui/src/app/app.js

```diff
@@ -1195,7 +1195,8 @@ YUI.add('juju-gui', function(Y) {
           appState={this.state}
```

Comment:
All good: Jeff will ask to port the same changes to the new init.js and init/... renderer ;-)